### ISSUE TITLE
FEATURE SUGGESTION: Show active cases

### ISSUE THREAD

**Issue by gpykhanna1** [CSSEGISandData/COVID-19#569]
First -- a big thank you for setting up this site.  It has been an invaluable resource!

Second -- Can you have a toggle switch that lets us see the active cases by country (cumulative cases - recovered - deaths).  That is, show the circles (and their diameter) is proportional to the cumulative cases.  We should have a view that shows the same for active cases.  I think this would not only be useful information but also help people keep things in perspective.

**Comment by lazd** [comment]
Agreed. This data is very important, total cases to date is interesting, but does not give the viewer the information they need: how serious is the situation at this location right now?

Though it's not the same map style, [I've done exactly that here](https://blog.lazd.net/coronaglobe/#type=active).

**Comment by analyzewithpower** [comment]
You can calculate this. Confirm-Deaths-Recoveries

**Comment by gpykhanna1** [comment]
> Agreed. This data is very important, total cases to date is interesting, but does not give the viewer the information they need: how serious is the situation at this location right now?
> 
> Though it's not the same map style, [I've done exactly that here](https://blog.lazd.net/coronaglobe/#type=active).

That is great -- thank you.  I was hoping the Johns Hopkins portal can also show this.

**Comment by gpykhanna1** [comment]
> You can calculate this. Confirm-Deaths-Recoveries

Yes -- that is the calculation.  I can, of course, do the calculation!  My point was to have this portal show this.  This portal has soared in popularity (and deservedly so).  So why not put that feature in that portal.

**Comment by analyzewithpower** [comment]
Oh. You mean not on the raw data but on the dashboard itself?

Yeah, that would be useful. :)

**Comment by nguyandy** [comment]
I've added the number of active cases, as well as the option to omit China from the dataset since they skew the data quite a bit.

https://covid19.nguy.dev/

**Comment by gpykhanna1** [comment]
> I've added the number of active cases, as well as the option to omit China from the dataset since they skew the data quite a bit. You can also search/select a country for more details.
> 
> https://covid19.nguy.dev/

Thank you!  This is a great site!

**Comment by AaronWard** [comment]
``` pip install covidify ```

This package generates time series reports for global and by country cases
It also shows stats like currently infected

**Comment by ODrying** [comment]
Active cases = Currently infected (the blue charts).

https://solutions.targit.com/default.aspx?doc=vfs://Global/COVID-19/Covid-19%20JH%20data%20Map%20Accumulated%20Confirmed.xview

![2020-03-12_22-35-53](https://user-images.githubusercontent.com/61792255/76569184-c927b380-64b2-11ea-95bf-d4551a52e254.png)

**Comment by Bost** [comment]
https://t.me/corona_cases_bot

![image](https://user-images.githubusercontent.com/1174677/76629545-b78bed80-653e-11ea-83d6-a77dd1aa50fb.png)
